docs/Client_Libraries/0_go.md

```diff
@@ -0,0 +1,156 @@
+# Go
+
+Source code: [github.com/oras-project/oras-go](https://github.com/oras-project/oras-go)
+
+## Introduction
+
+The ORAS Go client library provides the ability to replicate artifacts between different [Targets](../#target).  
+Furthermore, the version `v2` is a registry client conforming [image-spec v1.1.0-rc.2](https://github.com/opencontainers/image-spec/releases/tag/v1.1.0-rc2) and [distribution-spec v1.1.0-rc1](https://github.com/opencontainers/distribution-spec/blob/v1.1.0-rc1/spec.md).
+
+Using the ORAS Go client library, you can develop your own push/pull experience:

```

Comment:
Suggest changing `your own push/pull experience` to `your own registry client` since push/pull is narrow.

Reply:
Updated